docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -1,4 +1,4 @@
-// These env variables are shared across the docs. Copy and use
+// These environmental variables are shared across the docs. Copy and use
```

Comment:
I don't know that I've heard them referred to as "environmental variables." I think "environment variables" is a more appropriate term.

Reply:
Oh that was my mistake. 🙈